### ISSUE TITLE
Update GKE cluster creation parameters

### ISSUE THREAD

**Issue by aarontp** [google/turbinia#1114]
From a report on Slack:
```
By default Workload identity creates a GKE cluster with GKE metadata server which does not support instance/name parameter. In order for a worker to work properly, the GKE cluster must be created with the parameter --workload-metadata=GCE_METADATA
```

**Comment by aarontp** [comment]
@wajihyassine Could you take a look at this one?

**Comment by wajihyassine** [comment]
Ah yes, this will be fixed once the celery k8s deployment PR has been merged, specifically [here](https://github.com/google/turbinia/pull/1091/files#diff-7cc20ee15614ae2d13650a838e0c20841f6114426112bdf410fd7f952d95fead)

**Comment by wajihyassine** [comment]
Fixed in #1091
